path/masks.py

```diff
@@ -5,7 +5,10 @@
 
 
 # from jaraco.functools
-def compose(*funcs):
+from typing import Any, Callable
+
+
+def compose(*funcs: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
I observe that this type signature has [been updated](/jaraco/jaraco.functools/commit/0b3538dbbf1b9beb6f859674aab65f146316d471) and [again](/jaraco/jaraco.functools/commit/7661d952d053e77f13bec534996a4c37c8eb9580) upstream, so this signature is now stale. At the very least, we should move these vendored functions into their own modules so they're easier to keep separate from novel logic.